pkg/util/kubernetes/kubelet/types_kubelet.go

```diff
@@ -92,11 +92,20 @@ type ResourceName string
 // Resources name
 const (
 	ResourceCPU              ResourceName = "cpu"
+	ResourceNvidiaGPU        ResourceName = "nvidia.com/gpu"
```

Comment:
It might be good to link to some docs where these resource names are specified.

Reply:
Unfortunately there isn't a good doc explicitly specifying resource names. The best thing is [k8s schedule GPUs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/) which I will add . These specific GPU resources seems to be what the [ksm gpu check](https://github.com/DataDog/datadog-agent/blob/3b44cfbfc6508da0b73d3b7beec6e46a8d454dde/pkg/collector/corechecks/cluster/ksm/kubernetes_state_transformers.go#L29) is also covering.